packages/glow/src/glow.js

```diff
@@ -232,16 +235,20 @@ export function parseSyntax(lines, lang, prefix = true) {
   // multi-line comment
   let comment
 
-  function endComment() {
-    html.push({ comment })
+  function endComment(partial) {
+    html.push({ comment, partial })
     comment = null
   }
 
-  lines.forEach((line, i) => {
+  for (let i = 0; i < lines.length; i++) {
+    let line = lines[i]
     if (!comment) {
-      if (comm_start.test(line)) {
+      const start_comm = comm_start.exec(line)
+      const end_comm = comm_end.exec(line)
+      if (start_comm && start_comm.index === 0 && (!end_comm || line.length === end_comm.index + end_comm[0].length)) {
```

Comment:
@maurice : please note that Nue codebase prefers loose equality:

https://github.com/nuejs/nue/blob/master/CONTRIBUTING.md#code-style